### ISSUE TITLE
Tasks not working with new embarks - Steam version

### ISSUE THREAD

**Issue by Das123** [DFHack/dfhack#3258]
I've found in both the Beta of DF and the stable DF that Tasks don't appear with new embarks. I have completely uninstalled everything (removing the directory etc) and then slowly added in prefs etc to test each one. It is only when DFHack is installed that the Tasks stop functioning.
https://youtu.be/bk5ZMsqmnU4

**Comment by myk002** [comment]
I'm not seeing this in a fresh world with a fresh embark, replicated as close I could to what you did in the video
![image](https://user-images.githubusercontent.com/977482/232629934-1cdcde7b-9d32-48df-bc4d-b612589b7c44.png)

Could I get a link to a zip of your savegame and mods to see what's going on inside your particular fort?

**Comment by Das123** [comment]
Sorry. I had my pauses around the wrong way in the end. Here is the end re-recorded...
https://youtu.be/knyFm2OKyJQ.

I'm not sure how to link my save-files.

**Comment by myk002** [comment]
You can zip up your `save/regionX` directory and your `mods` directory and upload them to Google Drive (or any other cloud service). Make sure they're shared with "anyone who has the link", and post the links here.

**Comment by myk002** [comment]
Here's a full screencap of an embark for what I see. Nothing appears materially different from yours, except that one has dwarves doing things and one doesn't.

https://user-images.githubusercontent.com/977482/232634271-6eceae67-c17b-4bfa-8499-e638f2efae1a.mp4

Do you perhaps have Dwarf Therapist running?

**Comment by Das123** [comment]
No. Just vanilla.

**Comment by Das123** [comment]
Thanks Myk. Here is a save-file.
https://drive.google.com/file/d/1HRL9m0fId9dbNv2WBKZxlOdjL_56t1DY/view?usp=sharing
I tried opening it in DF without DFHack installed and it still has the tasks issue so hopefully you can see something in it. :)

**Comment by myk002** [comment]
I'm still trying to figure out the cause, but toggling DFHack autolabor on, playing for a bit, and then turning it off appears to get things working again. This indicates to me that it might have something to do with the internal debug flag Dwarf Fortress uses to control the work details system, but I'm still confused how your game got into this state in the first place.

**Comment by Das123** [comment]
I just joined the Discord server in case that would be a faster way of communicating for you.
Windowed mode didn't make any difference. Enabling autolabor did indeed work. Disabling afterwards still allows the tasks to continue. :)
Interestingly, I checked in my %appdata% directory to see if there was anywhere else that may be impacted by Dwarf Fortress. I found the even though I aren't running it, Dwarf Therapist activates a log file when DF runs.
